Sources/SwiftFrameCore/Extensions/String+Extensions.swift

```diff
@@ -10,6 +10,12 @@ extension String {
     func formattedRed() -> String {
         CommandLineFormatter.formatWithColorIfNeeded(self, color: .red)
     }
+    
+    func ky_containsHTMLTags() -> Bool {
+        let regex = try! NSRegularExpression(pattern: "<(.*)>.*?|<(.*) />")
```

Comment:
I think one or two simple tests to verify this regex would be good :) like, I don't really see any obvious issues with it, just for the sake of testing

Reply:
I believe there is no need to have a space before the closing bracket `/>` e.g. `<br/>` is a valid html tag

Reply:
Good call, will do 👌